test/cpp/api/functional.cpp

```diff
@@ -1127,6 +1127,92 @@ TEST_F(FunctionalTest, Threshold) {
   }
 }
 
+TEST_F(FunctionalTest, Interpolate) {
+  {
+    // 1D interpolation
+    auto input = torch::ones({1, 1, 2});
+    auto options = InterpolateOptions()
+                       .size(std::vector<int64_t>{4})
```

Comment:
would we be able to use
```suggestion
                       .size({4})
```
:D